lib/modules/platform/bitbucket/index.ts

```diff
@@ -623,7 +623,18 @@ export async function addReviewers(
 
   const body = {
     title,
-    reviewers: reviewers.map((username: string) => ({ username })),
+    reviewers: reviewers.map((username: string) => {
+      const UUIDRegex = new RegExp(
```

Comment:
```suggestion
      const UUIDRegex = regEx(
```
use our regex helper. also very bad practice to construct same regex inside loops. 😉

Reply:
Thanks! I updated and moved. I also saw that the flags are an argument, so switched that around as well.